src/Psalm/Storage/ClassConstantStorage.php

```diff
@@ -8,11 +8,27 @@
 
 class ClassConstantStorage
 {
+    use CustomMetadataTrait;
+
+    /**
+     * @var ?CodeLocation
+     */
+    public $type_location;
```

Comment:
Isn't used yet, but I feel like it should be. Should I add a psalm-suppress and hope someone gets around to using it, or can someone point me to where it should be used?

Reply:
This feels like something that will be used to solve https://github.com/vimeo/psalm/issues/5589

Reply:
I think it could be used for, e.g. highlighting, where issues refer to the constant type as opposed to the constant itself.

Reply:
> I think it could be used for, e.g. highlighting, where issues refer to the constant type as opposed to the constant itself.

Yeah, I'm just not sure how to go about finding all the places that should be happening.

Reply:
I decided to suppress it for now, I expect to use it in the issues emitted for #5589.